specs/core/shapes/Dialog-shape.ttl

```diff
@@ -95,7 +96,7 @@
 	oslc:name "usage" ;
 	oslc:propertyDefinition oslc:usage ;
 	oslc:occurs oslc:Zero-or-many;
-	dcterms:description """An identifier URI for the domain specified usage of 
+	dcterms:description """An identifier URI for the domain specified usage of
 		this dialog. If a resource has multiple uses, it may designate the primary or default one that should be used with a property value of oslc:default."""^^rdf:XMLLiteral ;
```

Comment:
there should be no whitespace here

Reply:
my editor strips trailing space automatically. was it wrong? the diff shows that a leftover space was deleted

Reply:
The problem is that the use of triple quotes will include both the newline and any leading space on the subsequent line as spaces within the resulting string literal - and that's not what we want here. The problem was not introduced by this change from multiple files to a single file - it was introduced over 4 years ago in commit a2ce6b14cc399307ead7ac5fb9851c7ffe67ee31!

The fix is to avoid using multiple lines in the source - just have one long line of source, or if that is just too long, and if embedded newlines are acceptable in the result, use triple quotes but have NO leading spaces on subsequent lines within the literal.